### ISSUE TITLE
Datetime error

### ISSUE THREAD

**Issue by Seanny123** [ownaginatious/fbchat-archive-parser#17]
In both Python 2 and Python 3 master branch of this repository, I'm getting the following error:

```
  File "/usr/local/bin/fbcap", line 11, in <module>
    load_entry_point('fbchat-archive-parser==0.8.post4', 'console_scripts', 'fbcap')()
  File "/usr/local/lib/python3.5/dist-packages/fbchat_archive_parser/main.py", line 153, in main
    app.run()
  File "/usr/local/lib/python3.5/dist-packages/clip.py", line 652, in run
    self.invoke(self.parse(tokens))
  File "/usr/local/lib/python3.5/dist-packages/clip.py", line 634, in invoke
    self._main.invoke(parsed)
  File "/usr/local/lib/python3.5/dist-packages/clip.py", line 519, in invoke
    self._callback(**{k: v for k, v in iteritems(parsed) if k not in self._subcommands})
  File "/usr/local/lib/python3.5/dist-packages/fbchat_archive_parser/main.py", line 94, in fbcap
    fbch = parser.parse()
  File "/usr/local/lib/python3.5/dist-packages/fbchat_archive_parser/parser.py", line 88, in parse
    self._parse_content()
  File "/usr/local/lib/python3.5/dist-packages/fbchat_archive_parser/parser.py", line 114, in _parse_content
    self._process_element(pos, element)
  File "/usr/local/lib/python3.5/dist-packages/fbchat_archive_parser/parser.py", line 237, in _process_element
    parse_timestamp(e.text, self.use_utc, self.timezone_hints)
  File "/usr/local/lib/python3.5/dist-packages/fbchat_archive_parser/time.py", line 165, in parse_timestamp
    timestamp = timestamp.datetime
AttributeError: 'datetime.datetime' object has no attribute 'datetime'
```

**Comment by Seanny123** [comment]
Fixed this issue by just removing that statement...

**Comment by ownaginatious** [comment]
Sorry, that was due to a lack of unit tests around that particular part of the code, which I now fixed. That should have only happened if your Facebook locale isn't in English or the other two supported locales (Norwegian and German for now).

Could you try running this again with the newest stuff from `master`? It will likely crash again (gracefully this time) complaining that it doesn't know how to parse your timestamps. I'd like a sample timestamp to add support for it.

**Comment by Seanny123** [comment]
Oh! I had forgotten that my locale for my Facebook had been switched to Japan for a significant period of time! I'll make a pull request to solve this issue.

**Comment by ownaginatious** [comment]
Fixed in `0.8.post9`.
